01-import_and_filter.py

```diff
@@ -106,8 +106,8 @@ def run_filter(subject, run=None, session=None):
         raw.resample(config.resample_sfreq, npad='auto')
 
     # Prepare the pipeline directory in /derivatives
-    fpath_out = op.join(config.bids_root, 'derivatives',
-                        'mne-study-template', subject_path)
+    deriv_path = op.join(config.bids_root, 'derivatives', 'mne-study-template')
```

Comment:
I am not too happy to see the name "study-template" in the code base. Imagine I fork the project, make my updated pipeline for my study. Now I need to potentially edit files unnecessarily. I could live if this path postfix is in the config though.

Reply:
okay, will fix

Reply:
see https://github.com/mne-tools/mne-study-template/pull/48/commits/7a1168ca09201c4454c32c1600ea3be4ce91e8f3